flake.nix

```diff
@@ -15,20 +15,10 @@
       inputs.flake-utils.follows = "k-framework/flake-utils";
       inputs.nixpkgs.follows = "k-framework/nixpkgs";
     };
-    ethereum-tests = {
-      url = "github:ethereum/tests/6401889dec4eee58e808fd178fb2c7f628a3e039";
-      flake = false;
-    };
-    ethereum-legacytests = {
-      url =
-        "github:ethereum/legacytests/d7abc42a7b352a7b44b1f66b58aca54e4af6a9d7";
-      flake = false;
-    };
```

Comment:
I presume the logic here is that we're never actually running these tests in a Nix build of KEVM, and so can drop them from being inputs?

Reply:
Yep. All the data they needed (2 files) have been copied into teh repo.